src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx

```diff
@@ -13,101 +13,127 @@ redirects:
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Query your data**: This is an example of a query and how to select a data format.
+  </figcaption>
 
-For numbers, select if you want us to auto-format them, or chose the number of decimals you want to see.
+  For numbers, select if you want us to auto-format them, or chose the number of decimals you want to see.
 
-<img
-  title="customchart03_decimals.png"
-  alt="customchart03_decimals.png"
-  src={queriesnrqlCustomChartDecimals}
-/>
+  <img
+    title="Data format: Auto-format"
+    alt="Data format: Auto-format"
+    src={queriesnrqlCustomChartDecimals}
+  />
 
-### Customize the Y axis [#custom-y-axis]
+  <figcaption>
+    **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Query your data**: This is an example of a query and how to select the **auto-format** option.
+  </figcaption>
+</Collapser>
 
-**Line charts and area charts.**
+<Collapser
+    id="customize-y-axis"
+    title="Customize the Y axis"
+  >
 
-On line charts and area charts you can adjust the Y axis to display the data within certain values by setting a minimum and maximum value for the axis.
+  On line charts and area charts you can adjust the Y axis to display the data within certain values by setting a minimum and maximum value for the axis.
 
-If no customization option is selected, [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards) automatically displays the full Y axis from 0 to the top value plus a margin.
+  If no customization option is selected, [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards) automatically displays the full Y axis from 0 to the top value plus a margin.
 
-### Enable or disable the legend [#custom-legend]
+  For line charts, area charts, and histograms, you can disable or enable the legend.
 
-**Line charts, area charts, and histograms.**
+  <img
+    title="Query your data: Legend"
+    alt="Query your data: Legend"
+    src={queriesnrqlCustomChartLegend}
+  />
 
-For line charts, area charts, and histograms, you can disable or enable the legend.
+  <figcaption>
+    **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Query your data**: This is an example of a query and how to enable the **Legend** option.
+  </figcaption>
+</Collapser>
 
-<img
-  title="customchart04_y_legend.png"
-  alt="customchart04_y_legend.png"
-  src={queriesnrqlCustomChartLegend}
-/>
+<Collapser
+    id="bar-charts-pie-charts-tables"
+    title="Bar charts, pie charts, and tables"
+  >
 
-### Remove the other groups facet [#custom-others]
+  When [faceting](/docs/query-your-data/explore-query-data/dashboards/filter-new-relic-one-dashboards-facets) on bar charts, pie charts, or tables, and if the number of faceting on queries is larger than 2,000, the **Other groups** facet aggregates the rest of facets. With this customization you can select whether to see **Other groups**, or remove it.
```

Comment:
I think this sentence could be improved slightly by a small restructure. Something like "If the number of [facets](/docs/query-your-data/explore-query-data/dashboards/filter-new-relic-one-dashboards-facets) on a query is larger than 2,000 for bar charts, pie charts, or tables, the **Other groups** facet aggregates the rest of the facets." I notice we use the word "facet" 3 times in this sentence, but I wouldn't know a good alternative for any of them.

Also. the "remove it" at the end of the second sentence reads a little strange. I know it refers to the option (and thus, "it"), but could "it" be changed to "them"?

Reply:
I've modified the entire paragraph. Let me know if it improves or not. Thanks!